gmail-tester.js

```diff
@@ -72,6 +72,12 @@ async function _get_recent_email(credentials, token, options = {}) {
           if (part.parts) {
             parts = parts.concat(part.parts);
           }
+          
+          if (!part.body.data) {
+            // body part could be an attachment of type text/plain or text/html
+            // and the parsing code below will break, hence skipping here
+            continue;
+          }
 
           if (part.mimeType === "text/plain") {
             email_body.text = Buffer.from(part.body.data, "base64").toString(
```

Comment:
What are your thoughts on this? Isn't this a more readable format? 
```suggestion
          if (part.body.data) {
            if (part.mimeType === "text/plain") {
              email_body.text = Buffer.from(part.body.data, "base64").toString(
              
              (...)
```

Reply:
I'd prefer returning early for error (in this case - missing body data) but please make the final call as u're the code owner 🙏